pkg/monitoring/docker/docker_windows.go

```diff
@@ -0,0 +1,11 @@
+// +build windows
+
+package docker
+
+func ListContainers() (map[string]interface{}, error) {
+	return nil, ErrorNotImplementedForOS
+}
+
+func ContainerNameByID(_ string) (string, error) {
```

Comment:
Shouldn't this return `ErrorNotImplementedForOS ` as `err` as well?

Reply:
ContainerNameByID() only called by `processes_notwindows.go`
But I fixed it to make it looks better :)